frontend/src/components/housing/enquiry/EnquiryIntakeForm.vue

```diff
@@ -81,8 +81,23 @@ const formSchema = object({
 // Actions
 const confirm = useConfirm();
 const router = useRouter();
+const route = useRoute();
 const toast = useToast();
 
+const getBackButtonConfig = computed(() => {
+  if (route.query.activityId) {
+    return {
+      text: 'Back to my drafts and previous entries',
+      routeName: RouteName.HOUSING_SUBMISSIONS
+    };
+  } else {
+    return {
+      text: 'Back to Housing',
+      routeName: RouteName.HOUSING
+    };
+  }
+});
+
```

Comment:
This doesn't appear to be functional? I am unable to trigger the first conditional to get "Back to my drafts..." even when navigating to the form from a draft enquiry.